rollup.config.js

```diff
@@ -0,0 +1,28 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import uglify from 'rollup-plugin-uglify';
+import babel from 'rollup-plugin-babel';
+import yargs from 'yargs';
+
+import pkg from './package.json';
+
+const args = yargs.argv;
+const external = Object.keys(pkg.dependencies || {}).concat(
```

Comment:
Getting close! I think this should only include `react`. The other dependencies are internal and should not need to be provided by consumers.

Also, I am getting this warning:

```
(!) Missing global variable names
Use options.globals to specify browser global variable names corresponding to external modules
react (guessing '_react')
```

I would expect that this should be listed as 'React' in the `options.globals`